Puppetfile

```diff
@@ -6,7 +6,9 @@ moduledir File.join(File.dirname(__FILE__), 'modules')
 
 # Core modules used by 'apply'
 mod 'puppetlabs-service', '3.1.0'
-mod 'puppetlabs-puppet_agent', '4.25.0'
+mod 'puppet-openvox_bootstrap',
+    git: 'https://github.com/jpartlow/puppet-openvox_bootstrap.git',
```

Comment:
We should be able to remove this pin to Josh's fork now that the PR has been merged.

Might need a release to remove the `git:` ref entirely though